config.style

```diff
@@ -990,6 +990,15 @@ Layer{
 	SpatialObjectTypes: http://inspire.ec.europa.eu/schemas/ge-core/4.0:MappedFeature
 	Styles: GE_GeologicUnit_AgeOfRocks
 }
+Layer{
+	id: "GE_Borehole"
+	registry-id: "http://inspire.ec.europa.eu/layer/GE.Borehole"
+	tags: inspire production
+	Name: "GE.Borehole"
+	Title: en "Boreholes", de "Bohrlöcher"
+	SpatialObjectTypes: http://inspire.ec.europa.eu/schemas/ge-core/4.0:Borehole
+	Styles: GE_Borehole
+}
```

Comment:
@JohannaOtt In the config.style document you always need to add a layer block (as you have added here), and down at the bottom of this document (line 1137 or after), you also need to add a style block.

Reply:
@JohannaOtt I see you added the Style block in the following commit.